rllib/agents/ddpg/ddpg_torch_model.py

```diff
@@ -88,9 +88,16 @@ def __init__(self,
         # emulate behaviour of tanh activation used in DDPG and TD3 papers.
         # After sigmoid squashing, re-scale to env action space bounds.
         class _Lambda(nn.Module):
+            def __init__(self_):
+                super(_Lambda, self_).__init__()
+                self_.action_range = nn.Parameter(
```

Comment:
why not use `self.register_buffer` ? since it is not a parameter actually.

Reply:
Good idea ! I will make the change when I can.